tools/add_triage_signature.py

```diff
@@ -273,7 +296,9 @@ def build_description(self, url, cluster_md):
                         "status": cluster_md['status'],
                         "status_info": cluster_md['status_info'],
                         "OCP_cluster_id": cluster_md['openshift_cluster_id'],
-                        "username": cluster_md['user_name']}
+                        "username": cluster_md['user_name'],
+                        "operators": self.build_feature_description(cluster_md, self.isOlm),
+                        "features": self.build_feature_description(cluster_md, self.isGeneralFeature)}
```

Comment:
let's do this simpler
create a method that returns two lists, ``operators`` and ``general_features`` based on the ``isOlm`` function alone

Reply:
It's actually simpler this way, because in the way you suggested I'll need a temp storage for those lists because the entries still need to be distinct. Don't you agree?

Reply:
Not really, as an example:
```
In [14]: all_features = set(["single-node", "additional-ntp", "CNV", "LSO"])

In [15]: operators = set(feature for feature in all_features if isOlm(feature))

In [16]: general_features = all_features - operators

In [17]: operators
Out[17]: {'CNV', 'LSO'}

In [18]: general_features
Out[18]: {'additional-ntp', 'single-node'}
```